src/test/java/seedu/address/logic/commands/AddCommandTest.java

```diff
@@ -147,6 +147,22 @@ public void setPerson(Person target, Person editedPerson) {
         public void updateFilteredPersonList(Predicate<Person> predicate) {
             throw new AssertionError("This method should not be called.");
         }
+
+        @Override
+        public ObservableList<Person> getDetailedContactView() {
+            throw new AssertionError("This method should not be called.");
```

Comment:
Although this was the previous AB3 design, I think it will be good for us to change this error into a sentinel error, or just change the string into a constant to reduce duplication!